tools/export-docu-to-website.sh

```diff
@@ -74,12 +78,11 @@ if [ "$TYPE" == "local" ]; then
 fi
 
 if [ "$TYPE" == "publish" ]; then
-   # Generate HTML and push to GitHub pages
+    # Generate HTML and push to GitHub pages
     if [ ! -d "$WORKSPACE/../iceoryx-web" ]; then
         cd $WORKSPACE/../
         git clone $WEBREPO
     fi
     cd $WORKSPACE/../iceoryx-web
-    mkdocs gh-deploy --config-file ../iceoryx/mkdocs.yml --remote-branch $VERSION
+    mike deploy --branch main --config-file ../iceoryx/mkdocs.yml --push --update-aliases $VERSION $ALIAS
```

Comment:
who's mike :laughing:

Reply:
[Mike](https://github.com/jimporter/mike) is guy who's doing versioning for us 😁

Reply:
nice guy :grin:

Reply:
Maybe geoffrey and mike can become friends :grin: